exercises/variable-length-quantity/canonical-data.json

```diff
@@ -0,0 +1,185 @@
+          "input": 4278190080,
+          "expected": [143, 248, 128, 128, 0]
+        },
+        {
+          "description": "maximum 32-bit integer input",
+          "input": 4294967295,
+          "expected": [143, 255, 255, 255, 127]
+        }
+      ]
+    },
+    "multiple": {
+      "description": ["Encode a series of integers, producing a series of bytes."],
+      "cases": [
+        {
+          "description": "multiple single-byte values",
+          "input": [64, 127],
+          "expected": [64, 127]
+        },
+        {
+          "description": "two multi-byte values",
+          "input": [16384, 1193046],
+          "expected": [129, 128, 0, 200, 232, 86]
+        },
+        {
+          "description": "many multi-byte values",
+          "input": [8192, 1193046, 268435455, 0, 16383, 16384],
+          "expected": [192, 0, 200, 232, 86, 255, 255, 255, 127, 0, 255, 127, 129, 128, 0]
+        }
+      ]
+    }
+  },
+  "decode": {
+    "single": {
+      "description": ["Decode a series of bytes, producing a single integer."],
+      "cases": [
+        {
+          "description": "one byte",
+          "input": [127],
+          "expected": 127
+        },
+        {
+          "description": "two bytes",
+          "input": [192, 0],
+          "expected": 8192
+        },
+        {
+          "description": "three bytes",
+          "input": [255, 255, 127],
+          "expected": 2097151
+        },
+        {
+          "description": "four bytes",
+          "input": [129, 128, 128, 0],
+          "expected": 2097152
+        },
+        {
+          "description": "maximum 32-bit integer",
+          "input": [143, 255, 255, 255, 127],
+          "expected": 4294967295
+        },
+        {
+          "description": "incomplete sequence causes error",
+          "input": [255],
+          "expected": null
+        },
+        {
+          "description": "incomplete sequence causes error, even if value is zero",
+          "input": [128],
+          "expected": null
+        },
+        {
+          "description": "overflowing a 32-bit integer causes error",
+          "#": "Your track may exclude this test if it is reasonable to give implementations the freedom to encode/decode larger numbers.",
+          "input": [255, 255, 255, 255, 127],
+          "expected": null
+        }
+      ]
+    },
+    "multiple": {
+      "description": ["Decode a series of bytes, producing a series of integers."],
+      "cases": [
+        {
+          "description": "multiple values",
+          "input": [192, 0, 200, 232, 86, 255, 255, 255, 127, 0, 255, 127, 129, 128, 0],
+          "expected": [8192, 1193046, 268435455, 0, 16383, 16384]
+        }
+      ]
+    }
+  },
+  "decode_encode": {
+    "cases": [
+      {
+        "description": "Round-trip through decode then encode should give original bytes.",
+        "input": [192, 0, 200, 232, 86, 255, 255, 255, 127, 0, 255, 127, 129, 128, 0],
```

Comment:
Worse, upon review, this exact series of bytes and corresponding integers is *already tested* in both the `encode` and `decode` sections. So I seriously doubt there can be an implementation that would fail this test while passing the others - this case can add no value.